frontend/ui/src/pages/Inbox/ConversationListItem/index.tsx

```diff
@@ -88,8 +88,38 @@ const ConversationListItem = (props: ConversationListItemProps) => {
     }
   }, [active, conversation, currentConversationState]);
 
-  const lastMessageIcon = (conversation: Conversation) => {
+  const lastMessageIsText = (conversation: Conversation) => {
     const lastMessageContent = conversation.lastMessage.content;
+
+    if (typeof lastMessageContent === 'string') {
```

Comment:
This code should be in the rendering library, right?

Reply:
it is in the rendering lib, we just didn't use it. We should refactor the conversationListItem so it uses the render lib for every source, don't we @AitorAlgorta @chrismatix

Reply:
Do we really want to use the render lib also fro the preview of the message? I am not sure what is best

Reply:
I think we should use the render lib to show the preview of the message, that way it shows more of a description than just an icon

Reply:
@AitorAlgorta Picking up on your other comment: Doing it in the render library is simpler, because each renderer is split into a `mapper` and a `renderer`. The `SourceMessagePreview` can re-use the `mapper` part. This is reason why I did the split in the first place :) 

Furthermore, we can simplify the preview into "text" vs "something else" (images, data, audio, etc.) to make the components simpler. So we add two components and then a `SourceMessagePreview`, where every source uses its `mapper` and then maps text to the text component and anything else to the "something else" (data) component.